float32[] velo
---
float32[] merge_velo